src/Tizen.Applications.EventManager/Tizen.Applications/ApplicationEventReceiver.cs

```diff
@@ -0,0 +1,160 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications.EventManager
+{
+    /// <summary>
+    /// The ApplicationEventReceiver Class provides functions to add event handler.
+    /// </summary>
+    /// <example>
+    /// <code>
+    ///void OnReceived(object sender, ApplicationEventArgs e)
+    ///{
+    ///     if (e.EventName == "event.org.tizen.example.AppEventTestApp.Tizen.Mobile.AppEvent")
+    ///     LogUtils.Write(LogUtils.DEBUG, LOG_TAG, "On Received : " + e.EventName);
+    ///}
+    /// public class EventManagerSample
+    /// {
+    ///     /// ...
+    ///    ApplicationEventReceiver receiver = new ApplicationEventReceiver("event.org.tizen.example.helloworld.AppEvent");
+    ///    receiver.Received += OnReceived;
+    /// }
+    /// </code>
+    /// </example>
+    /// <since_tizen> 6 </since_tizen>
+    public class ApplicationEventReceiver : IDisposable
+    {
+        private bool disposedValue = false;
+        private string _eventName;
+        private IntPtr _eventHandle;
+
+        private EventHandler<ApplicationEventArgs> s_eventHandler;
+        private Interop.AppEvent.EventCallback s_callback;
+
+        /// <summary>
+        /// Initializer of class.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ApplicationEventReceiver(string eventName)
```

Comment:
Method has 1 parameter, 0 is documented.

Reply:
Done